backend/tests/functional/ipam/test_proposed_change_reconcile.py

```diff
@@ -58,12 +63,18 @@ async def test_step01_add_address(
         branch_1,
         new_address_1,
     ) -> None:
+        await services.service.component.refresh_heartbeat()
         proposed_change_create = await client.create(
             kind=InfrahubKind.PROPOSEDCHANGE,
             data={"source_branch": branch_1.name, "destination_branch": "main", "name": "add_address_pc"},
         )
         await proposed_change_create.save()
         proposed_change_create.state.value = "merged"  # type: ignore[attr-defined]
+        await services.service.cache.set(
+            key=f"workers:active:{ComponentType.API_SERVER}:worker:{WORKER_IDENTITY}",
+            value=Timestamp().to_string(),
+            expires=KVTTL.FIFTEEN,
+        )
```

Comment:
is this something we could do as part of test initialization somewhere or will we need to add it to individual tests/classes like this sometimes?

Reply:
Yes I will circle back to this. I didn't want to change it at a more "global" level for this PR if it would lead to other tests starting to fail. But I have a note of this to trigger the scheduler to start during test init (previously this would always happen in a delayed manner in the background and the tasks would fail with log messages that we never really saw).